apps/condo/domains/ticket/utils/index.js

```diff
@@ -52,9 +53,28 @@ const buildFullClassifierName = (classifier) => {
     ].filter(Boolean).join(' » ')
 }
 
+const buildEmptyLineDifferentLength = (long = false) => {
```

Comment:
No, this is not about ticket, this is about some generation utils. Not all documents are word docs with ___ instead of blanks